client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -1,65 +1,75 @@
 /**
- * External depe;ndencies
+ * External dependencies
  */
 import React from 'react';
+
 /**
  * Internal dependencies
  */
 import ProgressBar from 'components/progress-bar';
 import Rating from 'components/rating';
 import analytics from 'lib/analytics';
 
-/**
- * Constants
- */
-const REVIEW_URL = 'https://wordpress.org/support/view/plugin-reviews/';
-
-module.exports = React.createClass( {
+export default React.createClass( {
```

Comment:
Let's leave that for another time. `React.createClass` is the most used syntax throughout our components, which is why I chose not to turn it into class for now.